cms/src/main/java/uk/nhs/digital/arc/util/HighchartsInputConversionForArc.java

```diff
@@ -0,0 +1,36 @@
+package uk.nhs.digital.arc.util;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import uk.nhs.digital.common.util.json.JacksonJsonSerialiser;
+import uk.nhs.digital.common.util.json.JsonSerialiser;
+import uk.nhs.digital.ps.chart.AbstractHighchartsParameters;
+import uk.nhs.digital.ps.chart.HighchartsParameters;
+import uk.nhs.digital.ps.chart.input.DelegatingHighchartsInputParser;
+import uk.nhs.digital.ps.chart.input.HighmapsXlsxInputParser;
+import uk.nhs.digital.ps.chart.input.ScatterHighchartsXlsxInputParser;
+import uk.nhs.digital.ps.chart.input.SeriesHighchartsXlsxInputParser;
+import uk.nhs.digital.ps.chart.model.AbstractHighchartsModel;
+
+import javax.jcr.Binary;
+import javax.jcr.RepositoryException;
+
+public class HighchartsInputConversionForArc {
+
+    public String process(String type, String title, String yTitle, Binary fileData) throws RepositoryException {
+        DelegatingHighchartsInputParser parser = new DelegatingHighchartsInputParser(new HighmapsXlsxInputParser(),
+            new SeriesHighchartsXlsxInputParser(),
+            new ScatterHighchartsXlsxInputParser());
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+
+        JsonSerialiser jsonSerialiser = new JacksonJsonSerialiser(objectMapper);
+
+        final AbstractHighchartsParameters parameters = new HighchartsParameters(type, title, yTitle, fileData);
+        final AbstractHighchartsModel chart = parser.parse(parameters);
+        final String chartConfigJson = jsonSerialiser.toJson(chart);
+
+        return chartConfigJson;
```

Comment:
Is there a reason why you declared this variable then immediately returned it? Can you not just do `return jsonSerialiser.toJson(chart);`?